packages/appkit-utils/src/ConstantsUtil.ts

```diff
@@ -1,13 +1,18 @@
 import type { ChainNamespace } from '@reown/appkit-common'
 
 export const ConstantsUtil = {
-  WALLET_CONNECT_CONNECTOR_ID: 'walletConnect',
-  INJECTED_CONNECTOR_ID: 'injected',
-  WALLET_STANDARD_CONNECTOR_ID: 'announced',
-  COINBASE_CONNECTOR_ID: 'coinbaseWallet',
-  COINBASE_SDK_CONNECTOR_ID: 'coinbaseWalletSDK',
-  SAFE_CONNECTOR_ID: 'safe',
-  LEDGER_CONNECTOR_ID: 'ledger',
+  /* Connector IDs */
+  CONNECTOR_ID: {
```

Comment:
💯

Reply:
Since they are under `CONNECTOR_ID`, we don't need to use `*_ID` postfix imo

Reply:
Yep good idea